index.js

```diff
@@ -5,7 +5,16 @@ var isglob = require('is-glob');
 var pathDirname = require('path-dirname');
 
 module.exports = function globParent(str) {
-	str += 'a'; // preserves full path in case of trailing path separator
+	// replace braces/brackets sections with *
+	str = str.replace(/(^|[^\\])(\{([^{}]*?)}|\[([^\[\]]*?)\])/g, '$1*');
+	if (/[\}\}]$/.test(str)) str += '/';
```

Comment:
is the goal to basically trick is-glob into returning true for half of a brace/bracket/parens pattern?